wot/ModelReader.py

```diff
@@ -204,27 +211,28 @@ def readMaterial(self, element):
 
 		# @TODO: Material has more properties ...
 		for item in element:
+			item_text = item.text.strip()
 			if item.tag == 'fx':
-				material.fx = item.text.strip()
+				material.fx = item_text
 			elif item.tag == 'collisionFlags':
-				material.collisionFlags = item.text.strip()
+				material.collisionFlags = item_text
 			elif item.tag == 'materialKind':
-				material.materialKind = item.text.strip()
+				material.materialKind = item_text
 			elif item.tag == 'identifier':
-				material.ident = item.text.strip()
+				material.ident = item_text
 			elif item.tag == 'property':
-				if item.text.strip() == 'diffuseMap':
```

Comment:
Because paths to textures are (were?) save as .tga, but actual textures are saved as .dds